hooks/useMatchData.ts

```
@@ -1,120 +1,160 @@
-            const awayTeam = match.team2 ? cleanTeamName(match.team2) : "";
-
-            // Extract time from date if it exists
-            if (match.date && !match.time) {
-              if (match.date.includes("T")) {
-                const timePart = match.date.split("T")[1];
-                match.time = timePart.substring(0, 5); // Extract HH:MM
+          const matches = data.events.map((event) => {
+            let homeTeam = "";
+            let awayTeam = "";
+
+            if (event.name && event.name.includes(" at ")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)